client/src/home/add_button/AddButton.js

```diff
@@ -127,10 +150,19 @@ function AddItemForm(props) {
         </Button>
         <Button
           variant="primary"
-          onClick={() => {
-            postNewItem(cookies.token, itemName, itemId, weight, outByDate);
-            props.closeForm();
-          }}
+          
+            onClick={() => {
+              if(itemName.length > 0 && batchId.length > 0){
+                postNewItem(cookies.token, itemName, batchId, weight, outByDate);
+                props.closeForm();
+              }
+              else{
+                <Button.Text className="warning">One or more required feild is invalid</Button.Text>
```

Comment:
Does this show which fields are invalid?

Reply:
Ah my bad I thought I took that out. That was an error message if someone tried to click the submit button and didn't have a required field, but that's not needed anymore